src/libraries/System.Diagnostics.EventLog/src/System/Diagnostics/Reader/NativeWrapper.cs

```diff
@@ -321,10 +321,10 @@ public static object EvtGetEventInfo(EventLogHandle handle, UnsafeNativeMethods.
                 int error = Marshal.GetLastWin32Error();
                 if (!status)
                 {
-                    if (error == UnsafeNativeMethods.ERROR_SUCCESS)
+                    if (error == Interop.Errors.ERROR_SUCCESS)
                     { }
                     else
-                        if (error != UnsafeNativeMethods.ERROR_INSUFFICIENT_BUFFER)
+                        if (error != Interop.Errors.ERROR_INSUFFICIENT_BUFFER)
```

Comment:
Can we fix the braces in this branch. It doesn't conform to the coding guidelines.